### PR TITLE
Feature - configure cypress tests

### PR DESCRIPTION
**Pull request by rikhilrai**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -6,6 +6,14 @@ env:
   DOCKER_IMAGE: docker.pkg.github.com/${{ github.repository }}/divergent-console
   DOCKER_TAG: latest
 jobs:
+  test-app:
```
Comment (rikhilrai) [inline .github/workflows/deploy.yml]: configured this as a separate job in the deploy pipeline. ideally would move this to the pull-request pipeline to run against a local server

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -40,3 +40,12 @@ jobs:
...
             cd $RUNNABLE_DIRECTORY
             ./run.sh
+  test-app:
+    needs: deploy-app
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          working-directory: app
```
Comment (krishan711) [inline .github/workflows/deploy.yml]: would be nice to use the working-directory syntax from above

Comment (krishan711) [reply .github/workflows/deploy.yml]:
```
    defaults:
      run:
        working-directory: app
```

**Review by krishan711 — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -40,3 +40,12 @@ jobs:
             git clone --depth 1 $REPOSITORY_URL $RUNNABLE_DIRECTORY
             cd $RUNNABLE_DIRECTORY
             ./run.sh
+  test-app:
+    needs: deploy-app
+    runs-on: ubuntu-latest
```
Comment (rikhilrai) [inline .github/workflows/deploy.yml:45]:
```suggestion
    runs-on: ubuntu-latest
    defaults:
      run:
        working-directory: app
```

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (rikhilrai) [inline .github/workflows/deploy.yml]:
```suggestion
```

**Pull request merged**